pkg/lifecycle/lifecycle.go

```diff
@@ -449,8 +449,9 @@ type Rule struct {
 
 // Configuration is a collection of Rule objects.
 type Configuration struct {
-	XMLName xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
-	Rules   []Rule   `xml:"Rule"`
+	XMLName           xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
+	Rules             []Rule   `xml:"Rule"`
+	ExpiryRuleRemoved bool     `xml:"ExpiryRuleRemoved,omitempty" json:"-"`
```

Comment:
```suggestion
	ExpiryRuleRemoved *bool     `xml:"ExpiryRuleRemoved,omitempty" json:"-"`
```

Keep it pointer such that `omitempty` works

Reply:
oh yes. will change